src/transformState.js

```diff
@@ -5,7 +5,17 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    if (action.type === 'addProperties') {
```

Comment:
this is the case to use switch-case
why? we have the limited number of conditions — 3, as we have 3 action types